app/scripts/lib/oauth-client.js

```diff
@@ -44,8 +45,37 @@ define(function (require, exports, module) {
     getClientInfo: function getClientInfo(id) {
       return this._request('get', GET_CLIENT + id);
     },
+    /**
+     * Fetch user's active OAuth clients
+     *
+     * @param {String} accessToken
+     * @returns {Promise}
+     */
+    fetchClients: function fetchClients(accessToken) {
+      var request = {
+        accessToken: accessToken,
+        type: 'get',
+        url: this._oAuthUrl + CLIENT_TOKENS_API
+      };
 
-    // params = { assertion, client_id, scope }
+      return xhr.oauthAjax(request);
+    },
+    /**
```

Comment:
Nit: space above and below this function, same comments as above for object shorthand and using const.